README.md

```diff
@@ -80,11 +84,11 @@
       - [Creating Access Service Config](#creating-access-service-config)
       - [Creating New Access Service Manager](#creating-new-access-service-manager)
     - [Using Access Services](#using-access-services)
-      - [Creating a New Project](#creating-a-new-project)
-      - [Updating a Project](#updating-a-project)
+      - [Creating a new project](#creating-a-new-project)
+      - [Updating a project](#updating-a-project)
       - [Deleting a Project](#deleting-a-project)
-      - [Assigning Repository To Project](#assigning-repository-to-project)
-      - [Unassigning Repository From Project](#unassigning-repository-from-project)
+      - [Assigning repository to project](#assigning-repository-to-project)
```

Comment:
The letters-casing of this title is now different than the rest of the titles in the table of contents.